### PR TITLE
testing for unknown property style

### PR DESCRIPTION
**Pull request by jameschenn**
css box model

### CODE REVIEW

File: prep-css-box-model/style.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+button {
+  background-color: red;
+  color: white;
+  margin: 20px;
+  padding: 10px;
+  border-width: 3px;
+  border-style: double;
+  border-color: darkgray;
+}
+}
```
Comment (thebearingedge) [inline prep-css-box-model/style.css:14]: Looks like you have an extra curly brace in here!

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: prep-css-box-model/style.css (hunk 1 above)
Comment (jameschenn) [reply prep-css-box-model/style.css]: I saw while looking for the fix! Thanks!!

**Comment by jameschenn** [comment]
Updated image

css box model update

**Review by thebearingedge — APPROVED** [review]
Nice!

**Pull request merged**